packages/modules/src/MultichainValidationModule.ts

```diff
@@ -0,0 +1,102 @@
+import { UserOperation, ChainId } from '@biconomy/core-types'
+import { Logger, getUserOpHash } from '@biconomy/common'
+import { ethers } from 'ethers'
+import MerkleTree from 'merkletreejs'
+import { ECDSAOwnershipValidationModule } from './ECDSAOwnershipValidationModule'
+import { MULTICHAIN_VALIDATION_MODULE_ADDRESSES_BY_VERSION } from './utils/Constants'
+import { keccak256, arrayify, defaultAbiCoder, hexConcat, hexZeroPad } from 'ethers/lib/utils'
+import {
+  ECDSAOwnershipValidationModuleConfig,
+  ModuleVersion,
+  MultiChainUserOpDto
+} from './utils/Types'
+export class MultiChainValidationModule extends ECDSAOwnershipValidationModule {
```

Comment:
@AmanRaj1608 @tomarsachin2271 since it extends the ECDSAModule chainId becomes mandatory but in real sense this module itself doesn't need to maintain chainId as it will receive it as part of dto (array of objects - userop + chainId) in singUserOps method. 
what should I do?

Reply:
ok let me do that.